spec/Section 3 -- Type System.md

```diff
@@ -1388,13 +1435,21 @@ Union types have the potential to be invalid if incorrectly defined.
 2. The member types of a Union type must all be Object base types; Scalar,
    Interface and Union types must not be member types of a Union. Similarly,
    wrapping types must not be member types of a Union.
+3. An union type may declare that it implements one or more unique interfaces.
```

Comment:
[Typo]: Super nit picky, but this should be "A union" instead of "An Union". I thought you were right, but it sounded weird so I looked it up. [An/A Union Explanation](https://english.stackexchange.com/questions/266309/why-is-union-an-exception-to-the-a-an-rule#:~:text=Usually%20when%20a%20word%20starts,why%20union%20is%20an%20exception.)